portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,29 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 id="text">Alexandra Valdes</h1>
+
+      <div class="grid-container">
+
+        <div class="grid-child">
+            <img src="/images/Headshot Icon.png" width = 425>
+        </div>
+
+        <div class="grid-child">
+            <p><br><br><br><br><b>Welcome to my Portfolio</b><br>
```

Comment:
Just something to note, if more precise spacing is needed between this and the next line, instead of using `<br>`'s   a class or a style can be added instead with `margin`!